readme.md

```diff
@@ -157,6 +187,12 @@ Type: `Function`
 
 Task function.
 
+##### skip
+
+Type: `Function`
+
+Skip function.
```

Comment:
Should we show the different possibilities here? Otherwise it will be difficult for users to know what it does in the different scenario's.

> - if skip returns false or a Promise for false, continue execution.
> - if skip returns true or a Promise for true, skip with a generic message.
> - if skip returns a string or a Promise for a string, skip with a custom message.
> - if skip throws or returns a rejected promise, fail the entire build

After this, it's good to merge I guess 🎉  👍  ❤️